ldi-api/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/types/LdiAdapter.java

```diff
@@ -0,0 +1,16 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.types;
+
+import org.apache.jena.rdf.model.Model;
+
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import static be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter.InputObject;
+
+public interface LdiAdapter extends LdiComponent, Function<InputObject, Stream<Model>> {
```

Comment:
Maybe add some javadoc that explains why you would want to use this? Should we update the readme as well as we have input/transformer/output there?

Reply:
In the readme, I've for now referred to the JavaDoc. Of course, this requires the java doc to be present :)